app/models/match.rb

```diff
@@ -9,4 +9,9 @@ class Match < ActiveRecord::Base
   # has_one :westeros_three_deck
   # has_one :tides_of_battle_deck # optional
 
+  scope :by_players, lambda { |player_ids| includes(:players).where(players: { id: player_ids }) }
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use the new lambda literal syntax `->(params) {...}`.](https://www.codacy.com/app/leandrosthimoteo_2996/got-board-api/file/2347594679/issues/source?bid=3216569&fileBranchId=3216571#l12)